nautobot_golden_config/utilities/helper.py

```diff
@@ -71,6 +75,19 @@ def get_job_filter(data=None):
             f"The following device(s) {', '.join([device.name for device in devices_no_platform])} have no platform defined. Platform is required."
         )
 
+    if query.get("device_status_id", None):
```

Comment:
* `if query.get("device_status_id", None):` is equivalant to `if query.get("device_status_id"):`
* I would think this should happen before the empty check
* I am not really sure what this is doing?

Reply:
> ``if query.get("device_status_id", None):`` is equivalant to ``if query.get("device_status_id"):``

Correct, this should be adjusted to the latter you mentioned or ``if "device_status_id" in query``